oceanbasecloud/manifest.json

```diff
@@ -0,0 +1,52 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "a42252f6-63f8-4da8-bce9-c765f30e7771",
+  "app_id": "oceanbase-cloud",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "uninstallation": "README.md#Uninstallation",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitoring OceanBase Cloud clusters with Datadog",
+    "title": "OceanBase Cloud",
+    "media": [],
```

Comment:
We require 3 images at least for the Integratoin tile. These images are the ones displayed when a user clicks on your integration. We recommend images of the dashboard, and images that show value of your integration. No marketing material